dev-python/m2crypto/m2crypto-0.31.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 python3_5 python3_6)
```

Comment:
Did/could you try with python3.7?

Reply:
No, I don't have it on any of my boxen, I guess it will work, but upstream mentioned only 3.6 in the changelog.

Reply:
I ran tests with python3.7. It seems to work, I believe you can add it. 
`PYTHON_COMPAT=( python2_7 python3_{4..7} )`